crates/sui-types/src/base_types.rs

```diff
@@ -373,6 +376,7 @@ impl TxContext {
             sender: AccountAddress::new(sender.0),
             digest: digest.into_inner().to_vec(),
             epoch: epoch_data.epoch_id(),
+            epoch_timestamp_ms: epoch_data.epoch_start_timestamp(),
```

Comment:
Note that it would be trivial at this point to use this value as a salt for ObjectID generation, but @kchalkias mentioned that with the introductionof 32B IDs and Addresses, this is no longer necessary, so I'm just exposing the timestamp to move calls (salting all the IDs will no doubt introduce a bunch of churn to tests, etc).

Reply:
This makes sense to me.